view/theme/frio/templates/head.tpl

```diff
@@ -39,11 +39,15 @@
 <link rel="shortcut icon" href="{{$shortcut_icon}}" />
 <link rel="apple-touch-icon" href="{{$touch_icon}}"/>
 
-<meta name="apple-mobile-web-app-capable" content="yes" /> 
+<meta name="apple-mobile-web-app-capable" content="yes" />
+<script>
```

Comment:
In general, I would agree, but adding this code to any externally loaded file could prevent it to run before the first link is tapped. And the code compression helps reduce the load increase by the inline JS.